ansys_optical_automation/application/stack_convert_to_speos.py

```diff
@@ -0,0 +1,45 @@
+from opcode import stack_effect
+import os
+import tkinter as tk
+from tkinter import filedialog
+
+from ansys_optical_automation.post_process.dpf_stack import DpfStack
+
```

Comment:
```suggestion


```

Reply:
As suggested by the failed pre-commmit style automation check, please run "pre-commit run -a" at terminal to make sure following Python PEP 8 standard.